client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -196,3 +210,38 @@ export class HubConnection {
         };
     }
 }
+
+class Base64EncodedHubProtocol implements IHubProtocol {
+    private wrappedProtocol: IHubProtocol;
+
+    constructor(protocol: IHubProtocol) {
+        this.wrappedProtocol = protocol;
+        this.name = this.wrappedProtocol.name;
+        this.type = ProtocolType.Text;
+    }
+
+    readonly name: string;
+    readonly type: ProtocolType;
+
+    parseMessages(input: any): HubMessage[] {
+        // atob/btoa are browsers APIs but they can be polyfilled. If this becomes problematic we can use
```

Comment:
I can think of a few things we might do here:
- Polyfill on nodejs - I hate this option
- Dependency injection
- Build a nodejs version with different sources